sunpy/__init__.py

```diff
@@ -9,26 +9,43 @@
 Homepage: http://sunpy.org
 Documentation: http://docs.sunpy.org/en/stable/
 """
-from __future__ import absolute_import
+# Enforce Python version check during package import.
+# This is the same check as the one at the top of setup.py
+import os
+import sys
+
+__minimum_python_version__ = "3.6"
```

Comment:
Can't we read this from setup.cfg or something? If we change minimum to something else, we'll have to change in two places.